### DISCUSSION TITLE
Payload for script execution with skiovox breakout?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1170] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
This(https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Skiovox%20Breakout) looks like it will work for my chromebook, does anyone have some good payloads for this?

**Comment by S-PScripts** [comment]
woah i wonder who made that :trollface: 


OP closed discussion, set answer [TN might have some payloads]
